FastSimulation/Tracking/plugins/TrajectorySeedProducer.h

```diff
@@ -43,7 +43,7 @@ class TrajectorySeedProducer:
         const MagneticFieldMap* magneticFieldMap;
         const TrackerGeometry* trackerGeometry;
         const TrackerTopology* trackerTopology;
-
+	//const RecHitsSortedInPhi * ih;
```

Comment:
here and in other modified files, please cleanup the commented out code, unless there is a good reason to keep it.